src/modeldatabase.jl

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env julia
 
+#
+# Some wrapper function for external maths functions
+#
+@inline function mittleff(α,z)
```

Comment:
Looks great. I think we'll need a similar one for the 3-parameter mittleff calls though

Reply:
Oh you were doing it already :+1: